api/response/models.py

```diff
@@ -27,6 +27,7 @@ class Response(Base, Utility):
     question_type = Column(Enum(QuestionType), default="rate")
     response = Column(postgresql.ARRAY(sa.String()))
     created_date = Column(DateTime, nullable=False)
+    updated_date = Column(DateTime, nullable=True)
```

Comment:
@koitoror Is there any reason why this timestamp column has a different name?

Reply:
@rajeman : unless for uniformity only, let me fix!